drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -175,9 +175,16 @@ static void nrf5_rx_thread(void *arg1, void *arg2, void *arg3)
 
 static enum ieee802154_hw_caps nrf5_get_capabilities(const struct device *dev)
 {
-	return IEEE802154_HW_FCS | IEEE802154_HW_FILTER |
-	       IEEE802154_HW_CSMA | IEEE802154_HW_2_4_GHZ |
-	       IEEE802154_HW_TX_RX_ACK | IEEE802154_HW_ENERGY_SCAN |
+	return IEEE802154_HW_FCS |
+	       IEEE802154_HW_FILTER |
+#if !defined(CONFIG_NRF_802154_SL_OPENSOURCE) && \
+    !defined(CONFIG_NRF_802154_SER_HOST)
+	       IEEE802154_HW_CSMA |
```

Comment:
Just to clarify, CSMA in the drvier is NOT supported with the upstream (open source) driver?

Reply:
It is not supported and currently there are no plans for it to be.